guidance/library/__init__.py

```diff
@@ -20,35 +20,34 @@
 from ._video import gen_video, video
 
 __all__ = [
-    "select",
-    "string",
-    "token_limit",
-    "with_temperature",
+    "Tool",
+    "assistant",
+    "at_most_n_repeats",
     "audio",
-    "gen_audio",
     "block",
+    "call_tool",
     "capture",
+    "exactly_n_repeats",
     "gbnf_to_lark",
-    "lark",
-    "call_tool",
     "gen",
-    "regex",
+    "gen_audio",
     "gen_image",
+    "gen_video",
     "image",
     "json",
+    "lark",
+    "one_or_more",
     "optional",
-    "assistant",
+    "regex",
     "role",
-    "system",
-    "user",
-    "at_most_n_repeats",
-    "exactly_n_repeats",
-    "one_or_more",
+    "select",
     "sequence",
-    "zero_or_more",
+    "string",
     "substring",
-    "Tool",
-    "gen_video",
+    "system",
+    "token_limit",
+    "user",
     "video",
-    "lark",
```

Comment:
This is actually what got me started doing this - `lark` appears twice in this list.